.gitignore

```diff
@@ -1,2 +1,77 @@
-node_modules/
-npm-debug.log
+### Node template
```

Comment:
We really don't need all this stuff in the gitignore. Was there something in particular that was causing you issues and wanting to be added to the repo?